### PR TITLE
Jfb 10 final merge for this project

### PR DESCRIPTION
**Pull request by chingiz-mazhitov**

### CODE REVIEW

File: src/test/java/com/andersen/UserServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	@Test
+	void findUserById_Found() {
+
+		when(userRepository.findById(1)).thenReturn(Optional.of(user));
+		User result = userService.findById(1);
+
+		assertEquals(result, user);
+	}
+
+	@Disabled
```
Comment (Alexmansar) [inline src/test/java/com/andersen/UserServiceTest.java]: why this annotation?

File: src/test/java/com/andersen/UserServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	@Test
+	void findUserById_NotFound() {
+		assertNull(userService.findById(99));
+	}
+
+	@Disabled
```
Comment (Alexmansar) [inline src/test/java/com/andersen/UserServiceTest.java]: same

File: src/test/java/com/andersen/UserServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+		userService.delete(user.getId());
+
+		verify(userRepository).deleteById(user.getId());
+	}
+
+	@Test
+	void deleteUser_ThrowException() {
+		doThrow(new RuntimeException("Delete operation failed")).when(userRepository).deleteById(2);
+		assertThrows(RuntimeException.class, () -> userService.delete(2));
+
```
Comment (Alexmansar) [inline src/test/java/com/andersen/UserServiceTest.java]: do you have such logic in delete method?

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@chingiz-mazhitov Hi)) i have small comments, please fix it) or answer)

File: src/test/java/com/andersen/UserServiceTest.java (hunk 3 above)
Comment (chingiz-mazhitov) [reply src/test/java/com/andersen/UserServiceTest.java]: no I don't have it in delete method, the assignment was to test all methods in the biggest service class and I did it. Should I remove the deleteUser_ThrowException?

File: src/test/java/com/andersen/UserServiceTest.java (hunk 1 above)
Comment (chingiz-mazhitov) [reply src/test/java/com/andersen/UserServiceTest.java]: I just temporarily decided to disable some test methods

File: src/test/java/com/andersen/UserServiceTest.java (hunk 3 above)
Comment (Alexmansar) [reply src/test/java/com/andersen/UserServiceTest.java]: yes) you don't have this logic in real method, so its not needed)) just remove))

File: src/test/java/com/andersen/UserServiceTest.java (hunk 1 above)
Comment (Alexmansar) [reply src/test/java/com/andersen/UserServiceTest.java]: Not the best solution in a real project)))) or if you need to, just add a comment as to why it's disabled

**Review by Alexmansar — APPROVED** [review]
@chingiz-mazhitov Hi)) please merge it))

**Pull request merged**